runtime/doc/eval.txt

```diff
@@ -5451,6 +5451,9 @@ jobstart({cmd}[, {opts}])				*jobstart()*
 		  |on_exit|  : exit event handler (function name or |Funcref|)
 		  cwd      : Working directory of the job; defaults to
 		             |current-directory|.
+		  env      : A dict of strings to append (or replace see |clear_env|) to the
```

Comment:
this line is longer than the standard length, add a newline after the closing parenthesis.